### PR TITLE
Pull Request for #1: Practice Task -- Uyseok Lee

### PR DESCRIPTION
**Pull request by uyseoklee**
@DavidRitzwoller 

I took 1000 draws from a bivariate normal distribution using R. The script can be found [here](https://github.com/uyseoklee/template/blob/issue1_practice/analysis/source/mvn_draws/make_draws.R). I saved the output to a csv file left in the build folder. 

I then plotted the 1000 draws using R. The script can be found [here](https://github.com/uyseoklee/template/blob/issue1_practice/analysis/source/mvn_plot/plot_draws.R). 

I saved the graph as a pdf in the release folder. Here is a quick link to the [graph](https://github.com/uyseoklee/template/blob/issue1_practice/analysis/release/mvn_plot.pdf).

I appreciate all the help so far! I'll wait for your further comments.

### CODE REVIEW

**Comment by DavidRitzwoller** [comment]
Meant to put this here:

Thanks @uyseoklee. Typically we would like to save the plot in release into its own directory mirroring the structure of source. Check out ad-price-drivers for examples. In this case mwn-plots.pdf should go into release/mvn-plot etc.

**Comment by uyseoklee** [comment]
@DavidRitzwoller I committed the branch again with your fixes. For quick reference, here is links to the updated folder in [release](https://github.com/uyseoklee/template/tree/issue1_practice/analysis/release/mvn_plot), the updated [config_global.yaml](https://github.com/uyseoklee/template/blob/issue1_practice/analysis/config_global.yaml) file, and the updated [SConstruct](https://github.com/uyseoklee/template/blob/issue1_practice/analysis/SConstruct) file.

**Comment by DavidRitzwoller** [comment]
Thanks @uyseoklee! Looks good to me.  Have you tried running all the code in paper_slides?

**Comment by uyseoklee** [comment]
> Thanks @uyseoklee! Looks good to me. Have you tried running all the code in paper_slides?

I hadn't yet gotten to that section from the original practice task, should I work on completing that task now?

**Comment by DavidRitzwoller** [comment]
Sounds good! let me know when the steps of the original task are completed.

**Comment by uyseoklee** [comment]
Morning @DavidRitzwoller, if you have a chance today, could you email the dropbox folder with the data to finish the initial practice task?

**Comment by uyseoklee** [comment]
@DavidRitzwoller I updated the figure and table to reflect the data that you gave me from dropbox. Here are quick links to the [paper](https://github.com/uyseoklee/template/blob/issue1_practice/paper_slides/release/paper/paper.pdf) and [slides](https://github.com/uyseoklee/template/blob/issue1_practice/paper_slides/release/talk/slides.pdf). I'll wait for your feedback!

File: analysis/source/descriptive/descriptive.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 def main():
     CONFIG =  yaml.load(open('config_global.yaml', 'rU'))
-    data  = np.genfromtxt('%s/data.txt' % CONFIG['build']['prepare_data'], skip_header = 1)
+    data  = np.genfromtxt('data.txt', skip_header = 1)
```
Comment (DavidRitzwoller) [inline analysis/source/descriptive/descriptive.py]: Can you upload data.txt to your dropbox, then use rclone to bring it to build/prepare_data. Raw Data should never be stored in analysis. We store it on dropbox, then rclone it to build

**Comment by uyseoklee** [comment]
@DavidRitzwoller The data.txt file has been moved to the proper location (build folder) and the code I used to get the data from dropbox was 
`rclone copy remote:data.txt build/prepare_data` (after I set up rclone using `rclone config'.

**Comment by DavidRitzwoller** [comment]
Looks good @uyseoklee! Feel free to close this task and PR

**Pull request closed without merging**